ihp.cabal

```diff
@@ -105,6 +105,7 @@ common shared-properties
             , with-utf8
             , ihp-hsx
             , ihp-postgresql-simple-extra
+            , auto-update
```

Comment:
Is this related to the debounce?

Reply:
Yes the `Control.Debounce` module is part of that package

Reply:
Should we add a comment to indicate it?

Reply:
Good point, added 👍